Challenge01-AzureAPIforFHIR/ReadMe.md

```diff
@@ -45,6 +47,12 @@ Make sure you have completed the pre-work covered in the previous challenge: [Ch
    Install-Module AzureAD -RequiredVersion 2.0.2.4
    ```
 
+   * __Note__: If you are using a `*nix` platform (Mac or Linux), you will need to install the `AzureAD.Standard.Preview` module instead of `AzureAD`:
+   ```powershell
+   Register-PackageSource -Trusted -ProviderName 'PowerShellGet' -Name 'Posh Test Gallery' -Location https://www.poshtestgallery.com/api/v2/'
+   Install-Module AzureAD.Standard.Preview
```

Comment:
In my Powershell Core environment, I had to import the module after installing it. Perhaps adding that to the code block would increase clarity? 

`Import-Module AzureAD.Standard.Preview`

I think this is especially important because if a user is running Powershell on *nix, they are less likely to have experience in Powershell.

Reply:
Done!